CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Grafana Mimir
 
 * [BUGFIX] Upgrade Go to 1.22.9 to address [CVE-2024-34156](https://nvd.nist.gov/vuln/detail/CVE-2024-34156). #10097
+* [BUGFIX] Update module google.golang.org/grpc to v1.64.1 to address [GHSA-xr7q-jx4m-x55m](https://github.com/advisories/GHSA-xr7q-jx4m-x55m). #8717
```

Comment:
nice catch

Reply:
Yeah, the changelog checking script caught it.